front/components/SongCardGrid.tsx

```diff
@@ -6,7 +6,7 @@ import { Heading, VStack } from 'native-base';
 type SongCardGrid = {
 	songs: Parameters<typeof SongCard>[0][];
 	maxItemPerRow?: number,
-	heading?: string
+	heading?: JSX.Element,
```

Comment:
Why this change here ? Because the Translate JSX is supposed to give us a string

Reply:
je me pose la même question du coup

Reply:
OK for the reusability of the component, but for the text update the component would have been fine because the text parameter would have to change and thus re-render the component